events/internal/equeue_platform.h

```diff
@@ -35,7 +35,7 @@ extern "C" {
 // Try to infer a platform if none was manually selected
 #if !defined(EQUEUE_PLATFORM_POSIX)     \
  && !defined(EQUEUE_PLATFORM_MBED)
-#if defined(__unix__)
+#if defined(__unix__) || defined(__APPLE__) || defined(__MACH__)
```

Comment:
Thank you for your PR. The check for Mac OS X should be `(defined(__APPLE__) && defined(__MACH__))`

Reply:
Thanks! Just changed :)